lib/paper_trail.ex

```diff
@@ -18,55 +18,66 @@ defmodule PaperTrail do
   """
   def insert(changeset, options \\ [origin: nil, meta: nil, originator: nil, prefix: nil]) do
     repo = RepoClient.repo()
-    transaction_order = case RepoClient.strict_mode() do
-      true ->
-        Multi.new
-        |> Multi.run(:initial_version, fn %{} ->
-          version_id = get_sequence_id("versions") + 1
-          changeset_data = case Map.get(changeset, :data) do
```

Comment:
This condition is not required because Map.get can return a default value which if you provide changeset as the default it does the same thing as this condition.